tests/Facebook/InstantArticles/Utils/FileUtilsPHPUnitTestCase.php

```diff
@@ -60,7 +60,7 @@ public function loadDOMDocumentFromString($fileContent, $encoding = 'utf-8')
     {
         libxml_use_internal_errors(true);
         $document = new \DOMDocument('1.0');
-        $document->loadHTML('<?xml encoding="${$encoding}" ?>'.$fileContent);
```

Comment:
@everton-rosario 

Yes, the first line is basically taken directly from here, hehe!